### PR TITLE
chore(markdown): Update option docs

### PR DESCRIPTION
**Pull request by joostdecock**
I just wanted to add that new `hide()` method to the option docs, but then ended up in a bit of a rabbit hole and now am not so sure this makes sense.

Would appreciate a second opinion so pinging @nicholasdower and @SeaZeeZee

### CODE REVIEW

File: markdown/dev/reference/api/config/dependencies/en.md [hunk 1]
```diff
@@ -2,14 +2,39 @@
...
+
+Dependencies control the order in which parts get drafted, but are also used
+when users requests to [only draft some parts of a 
+pattern](/reference/api/settings/only).
+Behind the scenes, FreeSewing will draft all dependencies, and make sure to not
+render them if they were not requested.
+
+## Structure
+
+An plain object of `key`-`value` pairs that controls the order in which pattern
```
Comment (nicholasdower) [inline markdown/dev/reference/api/config/dependencies/en.md:18]:
```suggestion
A plain object of `key`-`value` pairs that controls the order in which pattern 
```

File: markdown/dev/reference/api/config/hide/en.md [hunk 2]
```diff
@@ -2,14 +2,26 @@
 title: hide
 ---
 
+The `hide` key in the pattern configuration file allow you to configure
+parts the should be hidden by default.
```
Comment (nicholasdower) [inline markdown/dev/reference/api/config/hide/en.md:6]:
```suggestion
parts that should be hidden by default.
```

File: markdown/dev/reference/api/config/hide/en.md [hunk 3]
```diff
@@ -2,14 +2,26 @@
...
+*Hidden* means that they will be drafted, but not rendered. This is
+ypically used for a base part on which other parts are built.
```
Comment (nicholasdower) [inline markdown/dev/reference/api/config/hide/en.md:8]:
```suggestion
typically used for a base part on which other parts are built.
```

File: markdown/dev/reference/api/config/hide/en.md [hunk 4]
```diff
@@ -2,14 +2,26 @@
...
+
+Note that hidden parts will be rendered when the user requests 
+to [only draft some parts of a pattern](/reference/api/settings/only) 
+and includes the hidden part(s).
+
+
+## Structure
+
+An array with strings that hold part names.
```
Comment (nicholasdower) [inline markdown/dev/reference/api/config/hide/en.md:17]:
```suggestion
An array of strings that holds part names.
```

File: markdown/dev/reference/api/config/measurements/en.md [hunk 5]
```diff
@@ -2,15 +2,23 @@
 title: measurements
 ---
 
+The `measurements` key in the pattern configuration file allow you to configure
```
Comment (nicholasdower) [inline markdown/dev/reference/api/config/measurements/en.md:5]:
```suggestion
The `measurements` key in the pattern configuration file allows you to configure
```

File: markdown/dev/reference/api/config/name/en.md [hunk 6]
```diff
@@ -2,8 +2,17 @@
...
+name](https://github.com/npm/validate-npm-package-name).
+
+## Example
+
 ```js
 name: "sorcha"
 ```
 
-A string with the name of your pattern.
+In this example, the pattern is named **Sorcha**.
```
Comment (nicholasdower) [inline markdown/dev/reference/api/config/name/en.md:18]: nit/question: does the name actually get capitalized for you?

Comment (joostdecock) [reply markdown/dev/reference/api/config/name/en.md:18]: No, this is misleading. It probably was just muscle memory.

**Comment by joostdecock** [comment]
I guess it's pretty obvious that there's no spell check in my editor :grin:

File: markdown/dev/reference/api/config/name/en.md (hunk 6 above)
Comment (joostdecock) [reply markdown/dev/reference/api/config/name/en.md]: Fixed

**Pull request merged**